data/transactions/logic/evalStateful_test.go

```diff
@@ -632,14 +228,14 @@ pop
 		Clawback:      txn.Txn.Receiver,
 	}
 	algoValue := basics.TealValue{Type: basics.TealUintType, Uint: 0x77}
-	ledger := makeTestLedger(
+	ledger := logictest.MakeLedger(
 		map[basics.Address]uint64{
 			txn.Txn.Sender: 1,
 		},
 	)
-	ledger.newApp(txn.Txn.Sender, 100, basics.AppParams{})
-	ledger.balances[txn.Txn.Sender].locals[100]["ALGO"] = algoValue
-	ledger.newAsset(txn.Txn.Sender, 5, params)
+	ledger.NewApp(txn.Txn.Sender, 100, basics.AppParams{})
+	ledger.NewLocal(txn.Txn.Sender, 100, "ALGO", algoValue)
+	ledger.NewAsset(txn.Txn.Sender, 5, params)
```

Comment:
Cleaner ways to set up complicated balance tables for testing.  This was, coincidentally, recently requested from an outside contributor.